client/components/content-containers/tcc-exam/edit/index.vue

```diff
@@ -33,7 +33,7 @@
         @update="$emit('updateSubcontainer', $event)"
         @delete="$emit('deleteSubcontainer', group, 'group')"
         :group="group"
-        :tes="tes"
+        :elements="elements"
```

Comment:
Should we handle backwards compatibility ? 🤔

Reply:
I don't think anybody is using this container anymore so I don't we need to worry about it. And also this is a prop for a component which is only used in this container. It passes it on to its child which is also only used here. So we should be good on any breaking changes.